kraken-build/src/kraken/std/python/buildsystem/pdm.py

```diff
@@ -132,6 +132,7 @@ def get_managed_environment(self) -> ManagedEnvironment:
 
     def update_lockfile(self, settings: PythonSettings, pyproject: TomlFile) -> TaskStatus:
         command = ["pdm", "update"]
+        logger.info("Run '%s'", " ".join(command))
         sp.check_call(command, cwd=self.project_directory)
```

Comment:
Nit: we would really benefit to provide a single function for all these calls ;)